packages/api/models/base.js

```diff
@@ -474,6 +462,12 @@ class Manager {
       { status: 'failed', error: errorify(err), isActive: false }
     );
   }
+
+  validate(item) {
```

Comment:
Refacted to add a `this.validate()` method, so that we don't need to specify the schema and removeAdditional properties everywhere that we performed validation.  Also makes it easier to invoke validation from other places (like the ES indexer).

Reply:
`validate` is a pretty generic name. I don't mind as long as it's a member function, but a JSDoc would help clarify what it's validating against.

Reply:
Added documentation in 6a35abcc1